compiler/noirc_evaluator/src/ssa/ssa_gen/context.rs

```diff
@@ -1043,10 +1046,22 @@ fn convert_operator(op: BinaryOpKind) -> BinaryOp {
 impl SharedContext {
     /// Create a new SharedContext for the given monomorphized program.
     pub(super) fn new(program: Program) -> Self {
-        let mut globals_context = GlobalsContext::default();
+        let globals_shared_context = SharedContext::new_for_globals();
+
+        let globals_id = Program::global_space_id();
+
+        // Queue the function representing the globals space for compilation
+        globals_shared_context.get_or_queue_function(globals_id);
+
+        let mut context = FunctionContext::new(
+            "globals".to_owned(),
+            &vec![],
+            RuntimeType::Brillig(InlineType::default()),
```

Comment:
What does making globals' runtime brillig affect?

Reply:
It doesn't affect anything. I just chose Brillig as all ACIR globals will be inlined anyway so I felt it was clearer to mark it Brillig